### ISSUE TITLE
Missing validator should have a better error message

### ISSUE THREAD

**Issue by nambrot** [hyperlane-xyz/hyperlane-monorepo#1753]
A dev got

```
Error: Invalid config at `.` Error { path: Path { segments: [] }, original: missing field `validator` }

Caused by:
   0: .: missing field `validator`
   1: missing field `validator`
```

They forgot to specify the validator key env vars. We should have a better error message for that

**Comment by mattiecnvr** [comment]
This is a duplicate of #1593 and cannot be solved independently. The specific error that is displayed is an error generated by the `config` crate parsing. The changes in #1776 will _slightly_ help by removing the confusing `path: Path { segments: [] }` bit, but we can't fix the rest without something from #1593.

**Comment by Bukky3** [comment]
How does this happen?
